swagger.yaml

```diff
@@ -340,6 +340,102 @@ paths:
               schema:
                 $ref: "#/components/schemas/Error"
 
+  /v1/vacationsRequests:
+    get:
+      operationId: listVacationRequests
+      summary: List vacation requests
+      description: Lists persisted vacation requests
+      tags:
+        - Vacations
+      parameters:
+        - name: personId
+          required: false
+          in: query
+          description: Filter by persons unique ID
+          schema:
+            type: integer
+            format: int32
+        - name: before
+          required: false
+          in: query
+          description: Retrieve vacation requests before given date.
+          schema:
+            type: string
+            format: date
+        - name: after
+          required: false
+          in: query
+          description: Retrieve vacation requests after given date.
+          schema:
+            type: string
+            format: date
+      responses:
+        '200':
+          description: List of vacation requests.
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/VacationRequest'
+        "default":
+          description: Invalid request was sent to the server.
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/Error"
+
+  /v1/vacationRequest:
+    post:
+      operationId: createVacationRequest
+      summary: Post a new vacation request
+      description: Add new vacation request
+      tags:
+        - Vacations
+      requestBody:
+        description: New vacation request
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/VacationRequest'
+      responses:
+        '200':
```

Comment:
According to HTTP standard, a request that results in creating something new, should response with status code 201.